core/services/ocr2/plugins/ocr2keeper/evm21/feed_lookup.go

```diff
@@ -0,0 +1,444 @@
+package evm
+
+import (
+	"context"
+	"crypto/hmac"
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"math/big"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/avast/retry-go/v4"
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/patrickmn/go-cache"
+)
+
+const (
+	BlockNumber         = "blockNumber" // valid for v0.2
+	FeedID              = "feedID"      // valid for v0.3
+	FeedIDHex           = "feedIDHex"   // valid for v0.2
+	MercuryHostV2       = ""
+	MercuryHostV3       = ""
+	MercuryPathV2       = "/client?"
+	MercuryPathV3       = "/v1/reports?"
+	MercuryBatchPathV3  = "/v1/reports/bulk?"
+	NotFound            = "404"
+	InternalServerError = "500"
```

Comment:
I dont think we need these we can just do `fmt.Sprint(http.StatusNotFound)` inline where we need this

Reply:
done